packages/beacon-node/src/network/discv5/worker.ts

```diff
@@ -48,7 +48,10 @@ const config = createBeaconConfig(workerData.chainConfig, workerData.genesisVali
 const discv5 = Discv5.create({
   enr: SignableENR.decodeTxt(workerData.enr, keypair),
   peerId,
-  multiaddr: multiaddr(workerData.bindAddr),
+  bindAddrs: {
+    ip4: (workerData.bindAddrs.ip4 ? multiaddr(workerData.bindAddrs.ip4) : undefined) as Multiaddr,
+    ip6: workerData.bindAddrs.ip6 ? multiaddr(workerData.bindAddrs.ip6) : undefined,
+  },
```

Comment:
discv5 5.0.0 now requires bindAddrs to be passed in like so.
So we thread this through to the cli layer.